aspnetcore/tutorials/getting-started-with-NSwag.md

```diff
@@ -112,7 +112,7 @@ You can take advantage of NSwag's code generation capabilities by choosing one o
 * In the **Outputs** area, click the **CSharp Client** checkbox. Depending on your project, you can also choose **TypeScript Client** or **CSharp Web API Controller**. If you select **CSharp Web API Controller**, a service specification rebuilds the service, serving as a reverse generation.
 * Click **Generate Outputs** to produce a complete C# client implementation of the *TodoApi.NSwag* project. To see the generated client code, click the **CSharp Client** tab:
 
-:::code language="csharp" source="web-api-help-pages-using-swagger/_static/v6-nswag-generated-todoclient.cs" range="17-46":::
+:::code language="csharp" source="web-api-help-pages-using-swagger/_static/v6-nswag-generated-todoclient.cs" range="19-55":::
```

Comment:
The range is updated because of the generated output is different:

BEFORE (live website):
![image](https://github.com/dotnet/AspNetCore.Docs/assets/28659384/27774033-0c68-42c9-8b82-81a734464330)


AFTER (local website):
![image](https://github.com/dotnet/AspNetCore.Docs/assets/28659384/6a2a64ac-fb85-4d2c-b05e-d050bd6e84e0)